pkg/generate/generator.go

```diff
@@ -38,7 +38,8 @@ type Generator struct {
 	typeRenames      map[string]string
 	// Instructions to the code generator how to handle the API and its
 	// resources
-	cfg *ackgenconfig.Config
+	cfg         *ackgenconfig.Config
+	stringStore map[ackmodel.StringIdentifier]string
```

Comment:
Hmm. As it currently stands, the list of strings is here https://github.com/aws/aws-controllers-k8s/pull/449/files#diff-bbfadd4328f26a3a50306ade20809a5d15a16a0803dc14d85463e0cce64cb4eaR27

I was kind of hesitant about whether it'd scale to have all custom strings strongly typed in the structs because the list seems to be growing and having them hard-coded works for both ack and crossplane. But making them part of `Config` makes sense, too, in order to have some per-resource flexibility. I think we can have the current set of strings as hard-coded default for `Config` object and overwrite what we read from `generator.yaml` (if any) onto that `Config` object. Let me know what you think after looking at the current list and I'll make the change.